File: web-frontend/Models/Patient.cs
using System;

namespace PawsCare.Web.Models
{
    public class Patient
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Species { get; set; }
        public string Breed { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string OwnerId { get; set; }
        public string OwnerName { get; set; }
        public string MicrochipNumber { get; set; }
        public string Notes { get; set; }
    }
}

File: api-server/models/Patient.js
const mongoose = require('mongoose');

const patientSchema = new mongoose.Schema({
    name: { type: String, required: true },
    species: { type: String, required: true },
    breed: { type: String },
    dateOfBirth: { type: Date, required: true },
    ownerId: { type: String, required: true },
    ownerName: { type: String },
    microchipNumber: { type: String },
    notes: { type: String },
    createdAt: { type: Date, default: Date.now },
    updatedAt: { type: Date, default: Date.now }
});

module.exports = mongoose.model('Patient', patientSchema);
